SourceX/miniwin/misc.cpp

```diff
@@ -177,7 +181,11 @@ bool SpawnWindow(LPCSTR lpWindowName, int nWidth, int nHeight)
 #ifdef USE_SDL1
 		SDL_Log("upscaling not supported with USE_SDL1");
 #else
+#ifndef __SWITCH__
 		renderer = SDL_CreateRenderer(window, -1, SDL_RENDERER_PRESENTVSYNC | SDL_RENDERER_ACCELERATED);
+#else
+		renderer = SDL_CreateRenderer(window, 0, SDL_RENDERER_PRESENTVSYNC | SDL_RENDERER_ACCELERATED);

```

Comment:
https://github.com/carstene1ns/switch-sdl2-demo/blob/master/src/main.c#L73

Reply:
That person doesn't know what he/she is doing.

Let me test this myself and report back in a day or two.